stable/prometheus-mysql-exporter/templates/_helpers.tpl

```diff
@@ -30,13 +30,3 @@ Create chart name and version as used by the chart label.
 {{- define "prometheus-mysql-exporter.chart" -}}
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
-
-{{/*
-Secret name for cloudsql credentials
-*/}}
-{{- define "prometheus-mysql-exporter.cloudsqlsecret" -}}
```

Comment:
You should have kept it but changed as I suggested. It is still used in the chart.